src/averageAges.js

```diff
@@ -36,8 +40,21 @@ function calculateMenAverageAge(people, century) {
  *
  * @return {number}
  */
+
+function isHaveChildren(people, womenName) {
+  return people
+    .map(person => person.mother === womenName)
+    .includes(true);
```

Comment:
`Map` is not effective here - as you iterating all people elements

Consider using method that allows to check if people has at least one element for given condition